2_adventures/001/xcmer/05/hangman.rb

```diff
@@ -0,0 +1,84 @@
+require 'set'
+
+DICT_PATH = '/usr/share/dict/words'
+
+def get_words_count(dict)
+  count = 0
+  File.open(dict) {|f| count = f.read.count("\n") }
+end
+
+def get_random_word(dict, total_words)
+  number = Random.rand(total_words)
+  current = 0
+
+  File.foreach(dict) {|x|
+    if current == number
+      return x.chomp.downcase
+    end
+
+    current += 1
+  }
+end
+
+def print_word(word_letters, known_letters)
+  word_letters.each{|l|
+    if known_letters.include? l
+      print l
+    else
+      print '_'
```

Comment:
You unconditionally print, so why is it in a conditional branch (or rather in all the conditional branches)?  You want to simply print the result of the if statement.